extensions/wikia/Email/templates/temporaryPassword.mustache

```diff
@@ -3,8 +3,11 @@
 	<tr>
 		<td class="widest">
 			<p>{{passwordIntro}}</p>
+			{{# username}}
+				<p><pre>{{username}}</pre></p>
```

Comment:
Are username and password supposed to be surrounded in `<pre>` tags? The [designs](https://docs.google.com/document/d/1OM6xL0SNvkCyr7if8bwslFIeZWwwjW1Sdke2Y1A_s5A/edit) don't look like they call for them.